build/rust_toolchains.bzl

```diff
@@ -3,9 +3,9 @@ load("@rules_rust//rust:repositories.bzl", "rules_rust_dependencies", "rust_regi
 load("@rules_rust//tools/rust_analyzer:deps.bzl", "rust_analyzer_dependencies")
 load("//deps/rust/crates:crates.bzl", "crate_repositories")
 
-RUST_STABLE_VERSION = "1.90.0"  # LLVM 20
+RUST_STABLE_VERSION = "1.91.0"  # LLVM 20
```

Comment:
one tiny nit: Rust 1.91.0 is based on LLVM21, let's update the comment. https://releases.rs/docs/1.91.0/#internal-changes